### ISSUE TITLE
Compatibility with lakmeer's font

### ISSUE THREAD

**Issue by lagleki** [jackhumbert/zbalermorna-upright-font#4]
The current font uses space different from the https://github.com/lakmeer/zbalermorna one. It'd be nice to have characters in the same positions.

**Comment by jackhumbert** [comment]
Yeah! This one was aiming at being compatible with direct Lojban input, all via ligatures. I think I remember you mentioning on Discord that ligatures can't create all of the necessary symbols / doesn't have the correct precedence. 

[In my fork of lakmeer's repo](https://github.com/jackhumbert/zbalermorna), I'm working on a serif font with the same positions :) [there's a demo here (still a WIP)](https://jackhumbert.github.io/zbalermorna/test/index.html)!

**Comment by lagleki** [comment]
I just use a [slighly modified] lakmeer's algorithm to transcript Latin orthography into lakmeer's mapping. So being able to type just normal symbols and get zlm as a result would be awesome but i have to use https://framagit.org/peppercarrot/tools/blob/zbalermorna/utils/zbalermorna/piper.js as of now.

**Comment by jackhumbert** [comment]
Ah cool! It may be possible to have a font with the unicode mapping converted into another one with support for direct input via advanced ligatures. Thanks for linking the script - that's really helpful!

**Comment by lagleki** [comment]
> In my fork of lakmeer's repo, I'm working on a serif font with the same positions

it'd be super cool to have Comic-like font similar to the title in https://www.peppercarrot.com/0_sources/ep29_Destroyer-of-Worlds/low-res/en_Pepper-and-Carrot_by-David-Revoy_E29P00.jpg with large serifs that would give the feeling of fantasy-like font.

There are https://framagit.org/peppercarrot/fonts/blob/master/Latin/sources/PepperCarrot.sfd and Lavi (https://framagit.org/peppercarrot/fonts/tree/master/Latin/sources) if one needs inspiration.

**Comment by jackhumbert** [comment]
I was thinking that as well! That should be easy to configure. I'll mess around with that :)
